proto/invocation.proto

```diff
@@ -13,7 +13,22 @@ import "google/protobuf/duration.proto";
 
 package invocation;
 
-// Next tag: 26
+enum RemoteCacheOption {
+  UNKNOWN_REMOTE_CACHE_OPTION = 0;
+
+  // Neither --remote_cache nor --remote_executor is set.
+  CACHE_OFF = 1;
+  // --remote_upload_local_results is unset.
+  LOG_UPLOAD_ON = 2;
+  // --remote_download_outputs is set to "toplevel".
+  TOP_LEVEL_CACHE = 3;
```

Comment:
Not sure I understand LOG_UPLOAD_ON - is that mutually exclusive with `TOP_LEVEL_CACHE` / `MINIMAL_CACHE` ?

Reply:
Discussed offline - going to create 3 fields: 
- upload_local_results_enabled: bool
- download_outputs_option: enum (none, all, toplevel, minimal)
- remote_execution_enabled: bool